internal/cli/bridge.go

```diff
@@ -54,61 +58,146 @@ func BridgeCMD(toolName string) *cli.Command {
+
+	flags := []cli.Flag{
+		&cli.BoolFlag{
+			Name:     "qr-code-snapshot",
+			Required: false,
+			Usage:    "Bool to take a local snapshot instead of reading from an image file for recovery",
+			EnvVars:  []string{"QR_CODE_SNAPSHOT"},
+		},
+		&cli.StringFlag{
+			Name:     "qr-code-image",
+			Usage:    "Path to an image containing a valid QR code for recovery mode",
+			Required: false,
+			EnvVars:  []string{"QR_CODE_IMAGE"},
+		},
+		&cli.StringFlag{
+			Name:  "api",
+			Value: "127.0.0.1:8080",
+			Usage: "Listening API url",
+		},
+		&cli.BoolFlag{
+			Name:    "dhcp",
+			EnvVars: []string{"DHCP"},
+			Usage:   "Enable DHCP",
+		},
+		&cli.StringFlag{
+			Value:   "10.1.0.254/24",
+			Name:    "address",
+			EnvVars: []string{"ADDRESS"},
+			Usage:   "Specify an address for the bridge",
+		},
+		&cli.StringFlag{
+			Value:   "/tmp/kairos",
+			Name:    "lease-dir",
+			EnvVars: []string{"lease-dir"},
+			Usage:   "DHCP Lease directory",
+		},
+		&cli.StringFlag{
+			Name:    "interface",
+			Usage:   "Interface name",
+			Value:   "kairos0",
+			EnvVars: []string{"IFACE"},
+		},
+	}
+
+	flags = append(flags, cmd.CommonFlags...)
+
 	return &cli.Command{
 		Name:        "bridge",
-		UsageText:   fmt.Sprintf("%s %s", toolName, "bridge --network-token XXX"),
+		UsageText:   fmt.Sprintf("%s %s", toolName, "bridge --token XXX"),
 		Usage:       usage,
 		Description: fmt.Sprintf(description, toolName, toolName, toolName),
-		Flags: []cli.Flag{
-			&cli.StringFlag{
-				Name:     "network-token",
-				Required: false,
-				EnvVars:  []string{"NETWORK_TOKEN"},
-				Usage:    "Network token to connect over",
-			},
-			&cli.StringFlag{
-				Name:     "log-level",
-				Required: false,
-				EnvVars:  []string{"LOGLEVEL"},
-				Value:    "info",
-				Usage:    "Bridge log level",
-			},
-			&cli.BoolFlag{
-				Name:     "qr-code-snapshot",
-				Required: false,
-				Usage:    "Bool to take a local snapshot instead of reading from an image file for recovery",
-				EnvVars:  []string{"QR_CODE_SNAPSHOT"},
-			},
-			&cli.StringFlag{
-				Name:     "qr-code-image",
-				Usage:    "Path to an image containing a valid QR code for recovery mode",
-				Required: false,
-				EnvVars:  []string{"QR_CODE_IMAGE"},
-			},
-			&cli.StringFlag{
-				Name:  "api",
-				Value: "127.0.0.1:8080",
-				Usage: "Listening API url",
-			},
-			&cli.BoolFlag{
-				Name:    "dhcp",
-				EnvVars: []string{"DHCP"},
-				Usage:   "Enable DHCP",
-			},
-			&cli.StringFlag{
-				Value:   "10.1.0.254/24",
-				Name:    "address",
-				EnvVars: []string{"ADDRESS"},
-				Usage:   "Specify an address for the bridge",
-			},
-			&cli.StringFlag{
-				Value:   "/tmp/kairos",
-				Name:    "lease-dir",
-				EnvVars: []string{"lease-dir"},
-				Usage:   "DHCP Lease directory",
-			},
+		Flags:       flags,
+		Action:      bridge,
+	}
+}
+
+func stringsToMultiAddr(peers []string) []multiaddr.Multiaddr {
```

Comment:
This is taken verbatim from edgevpn, if you think the implementation makes sense, then I will do some refactoring in edgevpn to expose this function instead of copying here